### DISCUSSION TITLE
Disable "Turn on administrative access" and "Limited access"

### DISCUSSION THREAD

**Discussion by katodo** [cockpit-project/cockpit#21036] [Q&A] [answered]
Hi all, I use Cockpit on a device where I have different users with different permissions.

When I log in, especially with the user with the most limited permissions, I would like to remove the  "Turn on administrative access" banner and "Limited access" button from the home page.

This user can't become "administrator" and I don't like him seeing this banner.

Is there any option in the configuration file? I can't find anything in the documentation.

**Comment by martinpitt** [comment, marked as answer]
You can create a [manifest override](https://cockpit-project.org/guide/latest/packages.html#package-manifest-override) to disable all privileged bridges:
```
$ cat /etc/cockpit/shell.override.json
{ "bridges": [] }
```

That will disable the superuser indicator. It won't disable the "limited access mode" banner, though.

**Comment by katodo** [comment]
ok, i tried it and it actually works and it's already better than before. If i can't remove the banner that says "Web console is running in limited access mode." can i at least change the text of the message?

![image](https://github.com/user-attachments/assets/690c7d01-efc5-4db0-a250-06d8d1cb3f10)

**Reply by martinpitt** [reply]
Sorry, not with some config. It should be possible and to hard to hide the banner by default if there are no superuser bridges, as then it's useless. I filed #21134 about it.

**Reply by martinpitt** [reply]
See #21135 -- nice quick Friday afternoon hack!

**Reply by katodo** [reply]
Thanks! I see the patch is still under review, in the meantime I'll try it out by hand on my system.
